.github/workflows/codeql.yml

```diff
@@ -59,4 +49,4 @@ jobs:
       uses: github/codeql-action/analyze@v2
       with:
         category: "/language:${{matrix.language}}"
-      timeout-minutes: 30
+      timeout-minutes: 240
```

Comment:
Is this the only part that's hitting timeouts? Are the other portions OK? Should we also have a full-workflow timeout?

Reply:
This is currently the only part hitting the timeouts.  After we merge this I planned to re-review the times (hopefully without this being the limiting factor)